.travis.yml

```diff
@@ -19,13 +19,13 @@ cache:
 matrix:
   include:
   - jdk: openjdk7
-    script: mvn -q install -P examples
+    script: mvn -q install
```

Comment:
That will happen now. I've setup the pom in such a way that all modules and projects that can be build, will be build. 

Comes to mind I should probably push this change into #1154.